packages/connect-migrate/src/migrations/v1.6.0-transform.ts

```diff
@@ -138,12 +144,14 @@ function replace(
   qualifier?: string,
 ) {
   if (qualifier === undefined) {
-    root
-      .find(j.Identifier, { name: from })
-      .forEach((path) => (path.value.name = to));
+    // biome-ignore lint/complexity/noForEach: not alternative to forEach available
+    root.find(j.Identifier, { name: from }).forEach((path) => {
```

Comment:
I guess we could also do something like:

```tsx
for (const path of root.find(j.Identifier, { name: from }).paths()) {
  ...
```

Which would avoid the forEach. Not really sure it's worth the effort just to satisfy this lint rule 🤷

Reply:
This is already a very large diff, and who knows which linter we'll be using in a couple of months.